R/twSelectInput.R

```diff
@@ -43,7 +43,7 @@
 #' )
 #' }
 twSelectInput <- function(inputId, label, choices, selected = NULL,
-                          multiple = FALSE, selectize = TRUE, width = NULL, size = NULL,
```

Comment:
shiny::selectInput uses selectize = TRUE. Do we want to deviate from this here?

Reply:
Ohh hmm, my comment yesterday must have not worked. I was having a lot of problems with selective importing its custom files. I saw you trying to fix it, but I'm not sure how it is still loading.

Could you try loading example 4 with `selective = TRUE` and tell me if it happens to you?

Reply:
I have tried the following example and it works as expected from my side:
```R
library(shiny),
library(shiny.tailwind)

ui <- div(
  class = "grid grid-cols-2 gap-8",
  use_tailwind(),
  
  div(
    class = "border rounded-md bg-stone-200 shadow-md p-4",
    div(HTML("<code>shiny</code> inputs"), class = "font-bold text-4xl"),
    selectInput("id1", "Selectize", letters[1:3], multiple = TRUE, selectize = TRUE),
    selectInput("id2", "Select", letters[1:3], multiple = TRUE, selectize = FALSE)
  ),
  div(
    class = "border rounded-md bg-stone-200 shadow-md p-4",
    div(HTML("<code>shiny.tailwind</code> inputs"), class = "font-bold text-4xl"),
    twSelectInput("id3", "Selectize", letters[1:3], multiple = TRUE, selectize = TRUE),
    twSelectInput("id4", "Select", letters[1:3], multiple = TRUE, selectize = FALSE)
  )
)

shinyApp(ui, function(input, output) {})
```
![image](https://user-images.githubusercontent.com/15910496/176164470-a9e1a61b-87a1-45c4-b852-581d549f8e6b.png)